### PR TITLE
Added cvss_version field to vulnerability Note and Occurrence

### PR DESCRIPTION
**Pull request by wkozlik**
This is to provide information which of CVSS v3 or CVSS v2 were used to populate fields: cvss_score and severity.

### CODE REVIEW

File: proto/v1/cvss.proto [hunk 1]
```diff
@@ -157,3 +157,10 @@ message CVSS {
     IMPACT_NONE = 3;
   }
 }
+
+// CVSS Version.
+enum CVSSVersion {
+  CVSS_VERSION_UNSPECIFIED = 0;
+  CVSS_VERSION_2 = 2;
```
Comment (sherzberg-1) [inline proto/v1/cvss.proto:164]: Is there a reason you're starting from 2? This is just a proto field, so is there any place this value would get exposed as 2?

File: proto/v1/vulnerability.proto [hunk 2]
```diff
@@ -147,6 +147,11 @@ message VulnerabilityNote {
   // upstream timestamp from the underlying information source - e.g. Ubuntu
   // security tracker.
   google.protobuf.Timestamp source_update_time = 6;
+
+  // Output only. CVSS version used to populate cvss_score and severity.
```
Comment (sherzberg-1) [inline proto/v1/vulnerability.proto:151]: Why output only? On the note we store/populate it.

File: proto/v1/cvss.proto (hunk 1 above)
Comment (wkozlik) [reply proto/v1/cvss.proto]: Thanks for catching this. Corrected.

File: proto/v1/vulnerability.proto (hunk 2 above)
Comment (wkozlik) [reply proto/v1/vulnerability.proto]: Done.

**Review by sherzberg-1 — APPROVED** [review]

**Pull request merged**